### ISSUE TITLE
If Archive unlocked, Master Password can be changed without knowing the current one

### ISSUE THREAD

**Issue by exside** [buttercup/buttercup-desktop#625]
Title says it all, once a vault is unlocked and one tries to change the master password, there is no need to know the original master password, buttercup just allows to specify a new one, no questions asked. This is especially problematic with the missing auto-lock feature.

**Comment by perry-mitchell** [comment]
Good point @exside! We could easily prompt for the original password. I would say however that the damage is done if an _intruder_ were to be in front of an unlocked archive. Changing the master password would be the least of the owner's worries. Hence the Priority:High rating rather than critical. Autolock is definitely a must.

**Comment by fmommeja** [comment]
Fully agree @perry-mitchell !
This is high security concern and shouldn't be allowed IMHO because there's no time limited session.
